parse.y

```diff
@@ -16613,6 +16613,17 @@ rb_ruby_ripper_lex_state_name(struct parser_params *p, int state)
     return rb_parser_lex_state_name(p, (enum lex_state_e)state);
 }
 
+#ifdef UNIVERSAL_PARSER
+rb_parser_t *
+rb_ripper_parser_params_allocate(void)
+{
+    const rb_parser_config_t *config = rb_ruby_parser_config();
```

Comment:
`rb_global_parser_config` is ruby internal knowledge then parse.y should not know about that. Instead of that, please make the function to take `const rb_parser_config_t *config` as an argument like `rb_ruby_parser_allocate`?

Reply:
Thanks for the review.
I changed rb_ripper_parser_params_allocate to take const rb_parser_config_t *config as an argument.